Makefile

```diff
@@ -39,14 +39,13 @@ proto-lint:
 ###############################################################################
 ###                                Initialize                               ###
 ###############################################################################
-start-dev: install
```

Comment:
I kinda like this install step before running in case I forgot to build. Maybe you're right though and it should be a different step. 

Is there a downside to having the install here?

Reply:
Since this is just for demo, I guess we could just leave `install` command as it was for now. But I think in production level it should be a different step.

Reply:
Agreed. We may also have a different setup/command/script for different demos also.